interface/src/app/node/heavy_genes.tpl.html

```diff
@@ -0,0 +1,30 @@
+<div ng-if="!queryStatus">
+  <h4>
+    <div style="float: left">
+      <b>High-weight Genes: </b>{{genes.length}}
+    </div>
+    <div ng-if="genes.length > 0">
+      &nbsp({{hypoPercentage}}% Hypothetical)
```

Comment:
Also: since this is really a continuation of the `<h4>` above rather than a semantically different section, you could incorporate it directly using `<span>` tags, which are treated in-line rather than `<div>` which, by default is handled as a block element.

I would simplify lines 2-9 as follows:
```
<h4>
  <b>High-weight Genes</b>: {{genes.length}}
  <span ng-if="genes.length > 0">&nbsp;({{hypoPercentage}}% Hypothetical)</span>
</h4>
```

This has the added benefit of eliminating the need for `style="float: left"` on line 3.